booking/models.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+import uuid
+
+
+class Event:
+
+    def __init__(self, title, total_ticket_count, ticket_price,
+                 sale_end, sale_start,
+                 max_tickets_in_order=50,
+                 order_expiration_period=timedelta(minutes=30)):
+        self._id = uuid.uuid4().hex
+        self.title = title
+        self.total_ticket_count = total_ticket_count
+        self.ticket_price = ticket_price
+        self.sale_start = sale_start
+        self.sale_end = sale_end
+        self.max_tickets_in_order = max_tickets_in_order
+        self.order_expiration_period = order_expiration_period
+
+    @property
+    def id(self):
+        return self._id
+
+
+class TicketOrder:
+
+    def __init__(self, event, ticket_count, ticket_holder_name,
+                 billing_address, order_open_time):
+        self._id = uuid.uuid4().hex
+        self.event = event
+        self.ticket_count = ticket_count
+        self.status = 0
+        # Statuses:
+        # 0 - checkout, not paid
+        # 1 - paid
+        self.ticket_holder_name = ticket_holder_name
+        self.billing_address = billing_address
+        self.order_open_time = order_open_time
+
+    @property
+    def id(self):
+        return self._id
+
+    def pay(self):
+        self.status == 1
```

Comment:
There suppose to be one `=`, because pay is a verb which suggest that we want to mark that someone paid for ticket. `==` is checking if something is equall to so it should be used when we want to check if there was a payment done.

Reply:
Also, this is function which doesn't return anything.